docs/manual/webhooks/advanced-run-job.md

```diff
@@ -124,18 +131,22 @@ See [Path Examples](https://github.com/json-path/JsonPath#path-examples) in the
 repo for easy examples and inspiration.
 
 ### Template
-In a string template values are substituted with `${}`.
+Template Strings use the Groovy "GString" expansion syntax.
+
+Use `${VAR}` to expand to the value of the `VAR` variable, or `${VAR.key}` to access map entries.
+
+Given this example JSON Webhook event content, it will be provided as the variable `data`:
```

Comment:
To me this(along with the change on 144) reads like the content being under `data` is dependent on the example, when it would _always_ be under `data`.

Reply:
ok, i can clarify it